OpenRA.Mods.Common/Traits/BodyOrientation.cs

```diff
@@ -73,12 +73,16 @@ public BodyOrientation(ActorInitializer init, BodyOrientationInfo info)
 					return info.QuantizedFacings;
 
 				var qboi = self.Info.Traits.GetOrDefault<IQuantizeBodyOrientationInfo>();
-				var isb = self.HasTrait<ISpriteBody>();
 
 				// If a sprite actor has neither custom QuantizedFacings nor a trait implementing IQuantizeBodyOrientationInfo, throw
-				if (qboi == null && isb)
-					throw new InvalidOperationException("Actor" + self.Info.Name + "has a sprite body but no facing quantization."
-						+ " Either add the QuantizeFacingsFromSequence trait or set custom QuantizedFacings on BodyOrientation.");
+				if (qboi == null)
+				{
+					if (self.HasTrait<ISpriteBody>())
+						throw new InvalidOperationException("Actor" + self.Info.Name + "has a sprite body but no facing quantization."
```

Comment:
Please change the beginning to 
>("Actor " + self.Info.Name + "`` has a ...

for easier readability. Right now this results in "Actorlpst2has a sprite body".